### DISCUSSION TITLE
"Indexed DB" tables are missing & "Cache" also cleared automatically after PC crash due to unexpected shutdown

### DISCUSSION THREAD

**Discussion by Chris251290** [SafeExamBrowser/seb-win-refactoring#544] [Q&A]
Hi Team,

We are using SEB Version 3.3.2 for our Online Exam.

We are facing **Indexed DB crash issue** after the System power shutdown. Once the system powered on we have opened the SEB file, but our exam login page is not loading. **It is showing blank page**. **Cache also cleared automatically**. 

**Note**: We have not enabled the Cache clearing option anywhere in the SEB Configuration.

**Here the scenario which we have tested:**
1. Launched the exam using SEB file.
2. Entered Login credentials.
3. Attending the exam.
4. Internet connection got disconnected during the exam.
5. We are continuing the exam in Offline mode.
6. PC was shutdown due to power cut.
7. Powered on the PC once power came.
8. Launched the exam using SEB file without internet connection.
8. But, we are facing the blank screen issue.(Exam Login page is not appearing)
9. Then, We have opened the Developer Tool.
10. Checked the indexed DB.
11. **Indexed DB is empty now.**
12. **Cache also cleared automatically.**

Can you please check and tell us why this issue coming? & how to resolve the issue?

**Comment by dbuechel** [comment]
Does the issue reproduce with the latest release version? Please note that SEB does automatically clear the browser cache if set accordingly in the configuration: https://safeexambrowser.org/windows/win_usermanual_en.html#BrowserPane and https://safeexambrowser.org/windows/win_usermanual_en.html#ExamPane.
